src/mono/wasm/debugger/BrowserDebugProxy/DebugStore.cs

```diff
@@ -484,21 +486,21 @@ public TypeInfo(AssemblyInfo assembly, TypeDefinitionHandle typeHandle, TypeDefi
             this.type = type;
             methods = new List<MethodInfo>();
             Name = metadataReader.GetString(type.Name);
-            var namespaceName = "";
+            Namespace = "";
             if (type.IsNested)
             {
                 var declaringType = metadataReader.GetTypeDefinition(type.GetDeclaringType());
                 Name = metadataReader.GetString(declaringType.Name) + "/" + Name;
-                namespaceName = metadataReader.GetString(declaringType.Namespace);
+                Namespace = metadataReader.GetString(declaringType.Namespace);
```

Comment:
Will this work for nesting with more than 1 level? `class Foo { class Bar { class Abc {} }}`? Doesn't have to be fixed in this PR though